content/TrueCommand/Reports/Creation.md

```diff
@@ -14,43 +14,51 @@ The **Reports** page customizes system metrics charts for data analysis.
 
 ## Creating a Report
 
-Click *+ CREATE REPORT* to create a customizable report.
+Click **+ CREATE REPORT** to create a customizable report.
 Enter a report name and an optional description for the report.
 
 ![ReportsCreate](/images/TrueCommand/2.0/ReportsCreateReportNameDescription.png "Create a new report")
 
-Click *BROWSE WIDGETS* or *WIDGET* to add charts to the report.
+Click **BROWSE WIDGETS** or **WIDGET** to add charts to the report.
 
 ![ReportsCreateWidgetsAdd](/images/TrueCommand/2.0/ReportsCreateReportWidgets.png "Adding widgets to a new report")
```

Comment:
Replace image, remove top toolbar as it is incorrect for 2.1 and the TrueCommand Dashboard 'hover' box covers the screen header.